### PR TITLE
Portal

### PR DESCRIPTION
**Pull request by georgt99**
Adds my problem named `portal`.

I've not been able to test everything (like compiling the .tex), so there's probably some funny business going on.

### CODE REVIEW

**Comment by chistopher** [comment]
the makefile and .template folder are no symlinks. I'll commit the fix and then it builds. But test cases are >100MB. can you reduce the size/count of tests somewhat?

**Comment by georgt99** [comment]
Thanks for checking, symlinks were copied from one of the sample problems, but Windows probably doesn't maintain their symlinkiness.
Random test cases are now fewer and only one possible and impossible have the full 10^6 nodes, which brings it down to ~28MB total. Is that acceptable or are inputs with 10^6 lines a bad idea in general?

**Comment by chistopher** [comment]
problems with a linear or nlogn solution usually have around `2*10^5` input size. If you have 10^6 then you disallow some slow log factors and/or python

**Comment by georgt99** [comment]
alright, I went by the bounds in the Zahlen-slidedeck, but doesn't hurt to leave some buffer for snake-people 🐍

max size is now 2*10^5

**Comment by chistopher** [comment]
ok. i'll upload again

File: cp21/fun-contest/portal/executables/generator.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    '0\n'
+)
+Path('singleNodeInGraph.desc').write_text('starting room is the only room\n')
+
+# random possible instances
+for n in PROBLEM_SIZES:
+    input_text = ""
+    input_text += str(n) + '\n'
+    current_parent = 0
+    remaining_nodes = list(range(1, n))
```
Comment (PFischbeck) [inline cp21/fun-contest/portal/executables/generator.py:52]: I think this should be `range(1, n+1)`, otherwise there is one edge missing.

**Comment by chistopher** [comment]
updated on judge

**Pull request merged**